src/test/java/com/wikia/webdriver/testcases/renametool/RenameToolTests.java

```diff
@@ -0,0 +1,242 @@
+        .open()
+        .fillFormData("Chessky>Test", "Chessky>Test", "q")
+        .agreeToTermsAndConditions()
+        .submitChange();
+    Assertion.assertEquals(renameUserPage.getErrorMessage(),
+                           "This username contains non-alphanumeric characters.");
+  }
+
+  @Test(groups = {"renameTool_01"})
+  @Execute(asUser = User.QARENAME)
+  public void UserProvidesNoNewUserName_ErrorIsShown() {
+    SpecialRenameUserPage renameUserPage = new SpecialRenameUserPage(driver)
+        .open()
+        .fillFormData("", "", "q")
+        .submitChange();
+    Assertion.assertEquals(renameUserPage.getErrorMessage(), "This username is blank.");
+  }
+
+  @Test(groups = {"renameTool_01"})
+  @Execute(asUser = User.QARENAMEDALREADY)
+  public void UserAlreadyRenamedMessageShowed() {
+    SpecialRenameUserPage renameUserPage = new SpecialRenameUserPage(driver)
+        .open();
+
+    Assertion.assertStringContains(renameUserPage.getPageText(), "This account has already been "
+                                                                 + "renamed. As our ");
+  }
+
+  @Test(groups = {"renameTool_01"})
+  @Execute(asUser = User.QARENAME)
+  @RelatedIssue(issueID = "SUS-123", comment = "Ten test nie działą")
+  public void GoToHelpPage() {
+    SpecialRenameUserPage renameUserPage = new SpecialRenameUserPage(driver)
+        .open();
+    HelpPage helpPage = renameUserPage.goToHelpPage();
+
+    Assertion.assertEquals(helpPage.isHelpPageHeaderPresent(), true);
+  }
+
+  @Test(groups = {"renameTool_01"})
+  @Execute(asUser = User.QARENAME)
+  public void ConfirmationModalDecline_RedirectionToRenameTool() {
+    SpecialRenameUserPage renameUserPage = new SpecialRenameUserPage(driver)
+        .open()
+        .fillFormData("ChesskyTest", "ChesskyTest", "q")
+        .agreeToTermsAndConditions()
+        .submitChange();
+    new ConfirmationModalPage(driver).reject();
+    Assertion.assertEquals(renameUserPage.getHeaderText(), "Change a user's name");
+  }
+
+  @Test(groups = {"renameTool_01"})
+  public void NewUserCreateAndRenameDone() {
+    Credentials credentials = new Credentials();
+    String timestamp = Long.toString(DateTime.now().getMillis());
+    SignUpUser
+        user =
+        new SignUpUser("QARenameUser" + timestamp, credentials.email, "aaaa",
+                       LocalDate.of(1993, 3, 19));
+    UserRegistration.registerUserEmailConfirmed(user);
+    new WikiBasePageObject().loginAs(user.getUsername(), user.getPassword(), wikiURL);
+
+    SpecialRenameUserPage renameUserPage = new SpecialRenameUserPage(driver)
+        .open()
+        .fillFormData("NewUserName" + timestamp, "NewUserName" + timestamp, user.getPassword())
+        .agreeToTermsAndConditions()
+        .submitChange();
+    new ConfirmationModalPage(driver).accept();
+    Assertion
+        .assertEquals(renameUserPage.getSuccessBoxMessage(), "Rename process is in progress. The "
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Rename process is in progress. The " 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)